cirq/pasqal/pasqal_noise_model_test.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for pasqal_qubits."""
```

Comment:
Add copyright

Reply:
this module docstring seems unnecessary and wrong